editor.planx.uk/src/pages/Team/StartFromTemplateButton.tsx

```diff
@@ -7,28 +6,32 @@ import DialogContent from "@mui/material/DialogContent";
 import DialogTitle from "@mui/material/DialogTitle";
 import MenuItem from "@mui/material/MenuItem";
 import { useFormik } from "formik";
+import { useStore } from "pages/FlowEditor/lib/store";
 import React, { useState } from "react";
+import { useNavigation } from "react-navi";
 import { AddButton } from "ui/editor/AddButton";
 import SelectInput from "ui/editor/SelectInput/SelectInput";
 import InputLabel from "ui/public/InputLabel";
 
-export interface TemplateOverview {
+export interface TemplateOption {
```

Comment:
small nitty naming update here - this type describes the minimal template fields which are needed for the 'select' options in the modal ! "overview" feels like a word we might want to reserve for later - eg readme-esque description of the _whole_ template

Reply:
No issues here - this works for me! 👍